### ISSUE TITLE
Make sure the individual widget looks good on "6x7" phones.

### ISSUE THREAD

**Issue by adamwolf** [beeminder/beedroid#12]
_From @adamwolf on January 31, 2018 22:10_

See 	twenty13quarter's forum post.  (Waiting on clarification from user.)

**Comment by adamwolf** [comment]
I think this might have to do with minimum size requested.

**Comment by adamwolf** [comment]
The widget is resizable.  This may fix this.

**Comment by adamwolf** [comment]
The issue here has been fixed, but there are general widget appearance issues that need to be addressed in another issue.
